.gitattributes

```diff
@@ -30,3 +30,5 @@
 
 /.gitattributes export-ignore
 /.gitignore export-ignore
+
+/plugin-tests export-ignore
```

Comment:
Nit-pick, maybe put this exclude together with the other directories that contain tooling code (lines 8-10 above).